op-node/flags/flags.go

```diff
@@ -244,6 +244,12 @@ var (
 		EnvVars: prefixEnvVars("BETA_EXTRA_NETWORKS"),
 		Hidden:  true,
 	}
+	PostRegolithOverrideFlag = &cli.Uint64Flag{
+		Name:    "override.postregolith",
```

Comment:
Similar to op-geth PR, it would be really nice if we could just pick a name for this fork and use it from the start.

Reply:
naming it meadow